src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -112,6 +112,7 @@
 		mutableAudioOrVideoContainers.put(FormatConfiguration.THREEGPP2, new AudioVariantInfo(new THREEG2A(), FormatConfiguration.THREEGA));
 		// XXX WEBM Audio is NOT MKA, but it will have to stay this way until WEBM Audio is implemented.
 		mutableAudioOrVideoContainers.put(FormatConfiguration.WEBM, new AudioVariantInfo(new MKA(), FormatConfiguration.WEBA));
+		mutableAudioOrVideoContainers.put(FormatConfiguration.WMV, new AudioVariantInfo(new WMA(), FormatConfiguration.WMA));
```

Comment:
Why not, but it's only look as a quick patch to me.
As such files should be ASF specifications, that have audio and video MIME type, `audio/x-ms-asf`, and not WMV.
Apparently WP will not play such files as WMA or WMV:
https://github.com/videolan/vlc/blob/master/share/vlc.desktop.mimetypes#L26
https://docs.microsoft.com/en-us/windows/uwp/audio-video-camera/supported-codecs
Note: ASF is totally broken, not working, or has never worked at all, but it's not a big deal to fix it, at least by the way i did.

That was my 2 cents.